extensions/sql-assessment/src/telemetry.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Remove this stuff and the package if you're not using it currently

Reply:
I have brought this back.